### PR TITLE
feat(firebase): User class method to export db to a required format json

### PR DESCRIPTION
**Pull request by saratscheff**
Probé con la DB local y pude importarlo a firebase! :D
Este es el formato de la db en firebase:
```
{
  "pending" : {
    "alejandro0ramirez@gmail%2Ecom" : {
      "-LFFID__Z2BPeqQe01Up" : {
        "account_number" : "dsaf",
        "account_type" : "Cuenta Vista",
        "bank_name" : "Banco BICE",
        "user_email" : "alejandro0ramirez@gmail.com",
        "user_name" : "asdfasdf",
        "user_rut" : "123-1",
        "username" : "juan"
      },
      "-LFFIkX2EMjCQGOzMGb7" : {
        "account_number" : "asdgf",
        "account_type" : "Cuenta Ahorro",
        "bank_name" : "Banco Estado",
        "user_email" : "alejandro0ramirez@gmail.com",
        "user_name" : "asdfasdf",
        "user_rut" : "123-1",
        "username" : "sarachef"
      }
    },
    "asdf@asdf%2Ecl" : {
      "-LFG5le7MwdZ8JPxUOlf" : {
        "account_number" : "23rf",
        "account_type" : "Cuenta Ahorro",
        "bank_name" : "Banco BICE",
        "user_email" : "asdf@asdf.cl",
        "user_name" : "asdfasdf",
        "user_rut" : "234-1"
      }
    }
  },
  "user" : {
    "by_email" : {
      "alejandro0ramirez@gmail%2Ecom" : {
        "account_number" : "sdfgsdf",
        "account_type" : "Cuenta Ahorro",
        "bank_name" : "Banco de Crédito e Inversiones (Bci)",
        "user_email" : "alejandro0ramirez@gmail.com",
        "user_name" : "saldjf",
        "user_rut" : "181231234-3",
        "username" : "asd"
      },
      "oscar@platan%2Eus" : {
        "account_number" : "23423423",
        "account_type" : "Cuenta Vista",
        "bank_name" : "Consorcio",
        "user_email" : "oscar@platan.us",
        "user_name" : "Oscar",
        "user_rut" : "1231231-1"
      },
      "psaratscheff2@gmail%2Ecom" : {
        "account_number" : "8820357406",
        "account_type" : "Cuenta Corriente",
        "bank_name" : "Banco de Chile",
        "user_email" : "psaratscheff2@gmail.com",
        "user_name" : "Pedro Saratscheff",
        "user_rut" : "18.023.040-8",
        "username" : "juan"
      },
      "psaratscheff@gmail%2Ecom" : {
        "account_number" : "8820357406",
        "account_type" : "Cuenta Corriente",
        "bank_name" : "Banco de Chile",
        "user_email" : "psaratscheff@gmail.com",
        "user_name" : "Pedro Saratscheff",
        "user_rut" : "18023040-8",
        "username" : "saratscheff"
      }
    },
    "by_username" : {
      "asd" : "alejandro0ramirez@gmail.com",
      "juan" : "psaratscheff2@gmail.com",
      "sarachef" : "alejandro0ramirez@gmail.com",
      "saratscheff" : "psaratscheff@gmail%2Ecom"
    }
  }
}
```
Falta refinar en firebase el manejo de los username

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -6,6 +6,42 @@ class User < ApplicationRecord
...
 
   has_many :accounts, dependent: :destroy
+
+  def self.export_for_firebase
+    exported_json = "{
+  \"user\" : {
+    \"by_email\" : {"
+    User.where.not(confirmed_at: nil).each do |user|
+      exported_json += "
+      \"#{user.email.gsub('.','%2E')}\" : {
```
Comment (monkeyci) [inline app/models/user.rb:16]: Space missing after comma.

File: app/models/user.rb [hunk 2]
```diff
@@ -6,6 +6,42 @@ class User < ApplicationRecord
          :confirmable
 
   has_many :accounts, dependent: :destroy
+
+  def self.export_for_firebase
```
Comment (monkeyci) [inline app/models/user.rb:10]: Assignment Branch Condition size for export_for_firebase is too high. [35.36/25]<br>Method has too many lines. [33/15]

**Comment by saratscheff** [comment]
[migrar usuarios en RoR a firebase](https://trello.com/c/pM5dKUPY/50-migrar-usuarios-en-ror-a-firebase)

File: app/models/user.rb (hunk 1 above)
Comment (saratscheff) [reply app/models/user.rb:16]: TAMBIÉN HAY MONOCU ACÁ?! fooook jaja

**Pull request merged**